model/scratch/examples/scratchlgrsag/test_scratchlgrsag.py

```diff
@@ -0,0 +1,58 @@
+import os
```

Comment:
We need an `__init__.py` in `model/scratch/examples/scratchlgrsag/`

```console
$ touch model/scratch/examples/scratchlgrsag/__init__.py
```

Reply:
done